app/model/sdl/Abstract/Model.js

```diff
@@ -168,17 +169,27 @@ SDL.SDLModel = Em.Object.extend({
           return;
         }
 
+        var active_app_id = SDL.SDLController.model.appID
+        var app_model = SDL.SDLController.getApplicationModel(active_app_id)
+            
+        var video_width = SDL.SDLController.getApplicationModel(active_app_id).VideoConfigWidth
+        var video_height = SDL.SDLController.getApplicationModel(active_app_id).VideoConfigHeight
+
+        var marging_left = ((800 - video_width) / 2) 
```

Comment:
@LuxoftAKutsan I think you can use something like this to get actual work area size:
```
var view = document.getElementById('baseNavigation');
const view_width = view.offsetWidth;
const view_height = view.offsetHeight;
```

And then use it in your margin calculations:
```
var margin_left = ((view_width - video_width) / 2);
var margin_top = ((view_height - video_height) / 2);
```